eo2js/test/it/eo/win32-test.eo

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
```

Comment:
@skulidropek not a good place for such tests. Please check `eo2js/test/it/runtime-tests.test.js`. Here you can find excluded `sys/win32-tests`. Remove them for the `exclude` list and make sure they pass